acceptance/setup/aio/pre-suite/021_InstallAristaModule.rb

```diff
@@ -0,0 +1,18 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No arista hosts present" unless platforms.any? { |val| /eos/ =~ val }
```

Comment:
@joshcooper the need is to execute work on masters and agents, but only if one or more of the agents is of type `eos`. This pattern is the best that @LuvCurves and I could come up with to identify those constraints.